client/src/component/County/Audit/EndOfRound/FinalReviewPage.tsx

```diff
@@ -0,0 +1,170 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import CountyNav from 'corla/component/County/Nav';
+
+import { auditBoardSlice } from 'corla/selector/county/currentBallotNumber';
+
+import FinalReviewDialog from './FinalReviewDialog';
+
+import { Button, IButtonProps } from '@blueprintjs/core';
+
+import action from 'corla/action/';
+
+interface ReviewButtonProps {
+    cvr: JSON.CVR;
+    open: (cvr: JSON.CVR) => void;
+}
+
+const ReviewButton = (props: ReviewButtonProps) => {
+    const handler: IButtonProps['onClick'] = () => {
+        if (props.cvr.db_id != null) {
+            props.open(props.cvr);
+        } else {
+            alert('Sorry, this ballot cannot be re-audited.');
```

Comment:
The reason this code is/was here is I needed to add a guard when I was passing the ID itself after testing for its existence, because `db_id` is marked optional in the type system.

I now pass the entire CVR to the `open` callback, so there's not a need for it.